src/routes/router.ts

```diff
@@ -28,14 +28,10 @@ RootRouter.use(urlencoded({ extended: false }));
 RootRouter.use(json({
 	limit: "30mb", //set limit according to github doc https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-common-properties
 	verify: (_: Request, res: Response, buf) => {
-		try {
-			res.locals.rawBody = buf.toString();
-		} catch (e) {
-			// eslint-disable-next-line no-console
-			console.error("bgvozdev debugging just to make sure this never fails. \"try-catch\" will be removed after the experiment");
-		}
+		res.locals.rawBody = buf.toString();
```

Comment:
Wouldn't you want to use this on bodyparser, not json?

Also, it would make more sense to add it to the request and not res.locals.  We can update the Request type to have a `rawBody`

Reply:
You can update the request type in common.ts